pybamm/models/submodels/oxygen_diffusion/first_order_oxygen_diffusion.py

```diff
@@ -0,0 +1,79 @@
+#
+# Class for oxygen diffusion
+#
+import pybamm
+
+from .base_oxygen_diffusion import BaseModel
+
+
+class FirstOrder(BaseModel):
+    """Class for conservation of mass of oxygen. (First-order refers to first-order
+    expansion in asymptotic methods)
+    In this model, extremely fast oxygen kinetics in the negative electrode imposes
+    zero oxygen concentration there, and so the oxygen variable only lives in the
+    separator and positive electrode. The boundary condition at the negative electrode/
+    separator interface is homogeneous Dirichlet.
+
+    Parameters
+    ----------
+    param : parameter class
+        The parameters to use for this submodel
+    reactions : dict
+        Dictionary of reaction terms
+
+    **Extends:** :class:`pybamm.oxygen_diffusion.BaseModel`
+    """
+
+    def __init__(self, param, reactions):
+        super().__init__(param, reactions)
+
+    def get_coupled_variables(self, variables):
+
+        param = self.param
+        l_n = param.l_n
+        l_s = param.l_s
+        l_p = param.l_p
+        x_s = pybamm.standard_spatial_vars.x_s
+        x_p = pybamm.standard_spatial_vars.x_p
+
+        # Unpack
+        eps_0 = variables["Leading-order porosity"]
+        _, eps_s_0, eps_p_0 = [e.orphans[0] for e in eps_0.orphans]
+
+        # Diffusivities
+        D_ox_s = (eps_s_0 ** param.b) * param.curlyD_ox
+        D_ox_p = (eps_p_0 ** param.b) * param.curlyD_ox
+
+        # Reactions
+        sj_ox_p = sum(
+            reaction["Positive"]["s_ox"]
+            * variables["Leading-order " + reaction["Positive"]["aj"].lower()].orphans[
```

Comment:
can we use `"Average leading-order " + reac ...`. I think this is easier to read than doing orphans[0]